### PR TITLE
Polish

### PR DESCRIPTION
**Pull request by s12chung**
minor changes

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -1,3 +1,2 @@
 --color
---fail-fast
```
Comment (s12chung) [inline .rspec:2]: it's a personal preference. do you mind?

File: .ruby-version [hunk 2]
```diff
@@ -1 +0,0 @@
-2.2.2
```
Comment (s12chung) [inline .ruby-version:1]: think it conflicts if we ever decide to test multiple ruby versions. btw, this is useful for that in the future: https://github.com/thoughtbot/appraisal

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,14 +1,9 @@
...
+Bundler.require(:default, :test)
 Coveralls.wear!
 
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
+require 'inherited_class_var'
+
 RSpec.configure do |c|
   c.run_all_when_everything_filtered = true
-end
+end
```
Comment (s12chung) [inline spec/spec_helper.rb:9]: slightly cleaner. did you figure out coveralls? it took me too long to figure it out for `csv_row_model`

File: .rspec (hunk 1 above)
Comment (joel) [inline .rspec:2]: This boost time of CI in general CI ask for that, but doesn't matter here

File: .ruby-version (hunk 2 above)
Comment (joel) [inline .ruby-version:1]: Ok should be under `.git/info/exclude`

File: spec/spec_helper.rb (hunk 3 above)
Comment (joel) [inline spec/spec_helper.rb:9]: Ya maybe :confused: I don't know

File: Gemfile [hunk 4]
```diff
@@ -3,7 +3,11 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in inherited_class_var.gemspec
 gemspec
 
+gem "bundler", "> 1.3"
```
Comment (joel) [inline Gemfile:6]:
```
bundler --version
Bundler version 1.10.6
```

Comment (joel) [inline Gemfile:6]: Why this version?

File: inherited_class_var.gemspec [hunk 5]
```diff
@@ -11,22 +11,10 @@ Gem::Specification.new do |spec|
...
   spec.license       = 'MIT'
 
-  # Prevent pushing this gem to RubyGems.org by setting 'allowed_push_host', or
-  # delete this section to allow pushing this gem to any host.
-  # if spec.respond_to?(:metadata)
-  #   spec.metadata['allowed_push_host'] = 'TODO: Set to 'http://mygemserver.com''
-  # else
-  #   raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
-  # end
-
```
Comment (joel) [inline inherited_class_var.gemspec:21]: :+1:

File: inherited_class_var.gemspec [hunk 6]
```diff
@@ -11,22 +11,10 @@ Gem::Specification.new do |spec|
...
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   spec.bindir        = 'exe'
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
   spec.add_dependency 'activesupport',       '~> 4.2'
-
-  spec.add_development_dependency 'bundler', '~> 1.10'
-  spec.add_development_dependency 'rake',    '~> 10.0'
-  spec.add_development_dependency 'rspec',   '~> 3.3'
```
Comment (joel) [inline inherited_class_var.gemspec:31]: :+1:

File: spec/spec_helper.rb [hunk 7]
```diff
@@ -1,14 +1,9 @@
...
-
-begin
-  require 'pry'
-rescue LoadError
-end
-
+Bundler.require(:default, :test)
 Coveralls.wear!
 
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```
Comment (joel) [inline spec/spec_helper.rb:4]: Why this?

File: .rspec (hunk 1 above)
Comment (s12chung) [inline .rspec:2]: ah..... true. i'd rather let the machine do more work :p

File: Gemfile (hunk 4 above)
Comment (s12chung) [inline Gemfile:6]: not sure. it's just that all my gems allow this from 3 years ago.

File: spec/spec_helper.rb (hunk 7 above)
Comment (s12chung) [inline spec/spec_helper.rb:4]: https://gist.github.com/lroggendorff/3908654 -- default when you make a gem via `bundler`

File: spec/spec_helper.rb (hunk 3 above)
Comment (s12chung) [inline spec/spec_helper.rb:9]: yeah, it'd be nice to replace it in `csv_row_model` :P

File: Gemfile (hunk 4 above)
Comment (joel) [inline Gemfile:6]: :laughing: time to update

File: spec/spec_helper.rb (hunk 7 above)
Comment (joel) [inline spec/spec_helper.rb:4]: Isn't necessary nowadays :grin:

**Pull request closed without merging**